vbox_create.sh

```diff
@@ -166,7 +155,8 @@ function create_cluster_VMs {
   # Gather VirtualBox networks in use by bootstrap VM
   oifs="$IFS"
   IFS=$'\n'
-  bootstrap_interfaces=($($VBM showvminfo ${CLUSTER_PREFIX}bcpc-bootstrap \
```

Comment:
I wonder if it's less brittle to refer to `$(cat .vagrant/machines/<prefix>bcpc-bootstrap/virtualbox/id)` instead

Reply:
Hrm, that does look like a nice stable ID pointer. I usually prefer to use `vboxmanage` since it tends to be reasonably stable. I guess I just do not have as much faith in Vagrant's file layout stability?

Reply:
True. since this is vagrant internals I don't think there will be promises of file layout stability in here.